torch/fft/__init__.py

```diff
@@ -9,18 +9,233 @@
 # connects the torch.fft Python namespace to the torch._C._fft builtins.
 
 fft = _add_docstr(_fft.fft_fft, r"""
-fft(input) -> Tensor
+fft(input, n=None, dim=-1, norm=None) -> Tensor
 
 Computes the one dimensional discrete Fourier transform of :attr:`input`.
 
+Note:
+
+    The Fourier domain representation of any real signal satisfies the
+    Hermitian property: `X[i] = conj(X[-i])`. This function always returns both
+    the positive and negative frequency terms, even though for real inputs the
+    negative frequencies are redundant. :func:`torch.fft.rfft` returns the more
+    efficient "half-complex" representation where only the positive frequencies
+    are stored explicitly.
+
+Args:
+    input (Tensor): the input tensor
+    n (int, optional): FFT length. If given, the input will either be zero-padded
+        or trimmed to this length before taking the FFT.
+    dim (int, optional): The dimension along which to take the one dimensional FFT.
+    norm (str, optional): Normalization mode:
+        ``"forward"`` - the forward transform (:func:`~torch.fft.fft`) is normalized by ``1/n``,
+        ``"backward"`` - the backward transform (:func:`~torch.fft.ifft`) is normalized by ``1/n``,
```

Comment:
So what does calling `fft` with norm mode backward actually do? These statements are clearer but it's still odd to me that the backward description refers to a different function.

Reply:
If you do `ifft(fft(x, mode=mode), mode=mode)` then no matter what mode you use, the total normalizing factor of `1/N` is applied. What changes is which transform does the normalization. So, `fft(x, norm="backward")` is the unnormalized FFT and `ifft(x, norm="forward")` is also unnormalized.

Reply:
I think I get that and I understand where the documentation is coming from, but maybe the "backward" description could refer to its effect on the function being documented (unnormalized) and then add a follow-up sentence to include this perspective? Because as it is the effect of "backward" on fft is unspecified.